libs/remix-ui/static-analyser/src/lib/remix-ui-static-analyser.tsx

```diff
@@ -77,6 +84,13 @@ export const RemixUiStaticAnalyser = (props: RemixUiStaticAnalyserProps) => {
     return () => { }
   }, [state])
 
+  useEffect(() => {
```

Comment:
A useEffect without a dependency array will lead to memory leak because this event will be fired on every change to the component.

Reply:
added, please check